app/retail/templates/shared/cards.html

```diff
@@ -27,10 +27,8 @@
 {% endif %}
 {% if card_desc %}
 <meta name="twitter:description" content="{{card_desc}}" />
-<meta name="description" content="{{card_desc}}" />
```

Comment:
Why remove these meta tags?

Reply:
Multiple reasons:
* This file `cards.html` should be used only for social media meta data, not for page properties
* The meta description will be set already in https://github.com/gitcoinco/web/blob/f6e5b1e56ffd65945e517427a6b072b2c22c3ecc/app/retail/templates/shared/head.html#L78 The W3C validator reports the double output of meta description, see https://validator.w3.org/nu/?doc=https%3A%2F%2Fgitcoin.co%2F

Reply:
humm the problem is Im not sure if is used in the same way everywhere

Reply:
Okay, but a page meta description is wrong in `cards.html` file. The better way would to create a `shared/meta.html` with page meta informations and include the file everywhere.

Reply:
I don't found any Gitcoin page without meta description generated from `web/app/retail/templates/shared/head.html`. You?